src/language/__tests__/visitor-test.ts

```diff
@@ -893,6 +894,41 @@ describe('Visitor', () => {
     ]);
   });
 
+  it('visits only the specified `Kind` in visitorKeyMap', () => {
+    const visited: Array<[string, string, ReturnType<typeof getValue>]> = [];
```

Comment:
All other test use `Array<any>`.
If you want to improve typings it should be done in a separate PR and for the whole file.